src/wp-includes/l10n.php

```diff
@@ -1712,3 +1712,49 @@ function is_locale_switched() {
 
 	return $wp_locale_switcher->is_switched();
 }
+
+/**
+ * Translates the provided settings value using its i18n schema.
+ *
+ * @internal
```

Comment:
As per the [documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods), `@internal` is meant for adding some internal notes.

If you want to declare this private, you'd need to use `@access private`.

It needs to be added directly below the `@since` line

Reply:
I added both, so I guess I will remove `@internal and we should be good. Thanks for sharing the reference from docs.